src/main/java/it/pagopa/pagopa/apiconfig/service/IcaService.java

```diff
@@ -243,6 +268,40 @@ public List<CheckItem> verifyIca(MultipartFile file, Boolean force) {
         return checkItemList;
     }
 
+    // added to avoid sonar warning, we need to use tempFile to avoid to analyze hidden files and directories
+    @java.lang.SuppressWarnings({"javasecurity:S6096", "java:S5443"})
```

Comment:
Is it possible to use [this approach](https://rules.sonarsource.com/java/RSPEC-5445?search=Functional%20Interfaces%20should%20be%20as%20specialised) instead of suppressing the warning?

Reply:
The warnings are for the name selection of the tempFile, since we do not need a temp directory I don't think it is applicable